mdpow/workflows/base.py

```diff
@@ -173,3 +178,34 @@ def automated_project_analysis(project_paths, ensemble_analysis, **kwargs):
 
     logger.info('all analyses completed')
     return
+
+def guess_elements(universe_atom_group):
+    # input from dihedrals module
+    # universe_atom_group = universe.atoms
+    # universe from MDPOW FEP project results
+    names = universe_atom_group.names
+    masses = universe_atom_group.masses
+
+    guessed_elements = guessers.guess_types(names)
+    guessed_masses = np.array([guessers.get_atom_mass(n) for n in guessed_elements])
+
+    problem_cases = []
+    i=0
+    while i < len(masses):
+        if masses[i] != 0:
+            value = math.isclose(guessed_masses[i], masses[i], rel_tol=0.001)
+            value = not(value)
+        else:
+            value = False
+        problem_cases.append(value)
+        i+=1
```

Comment:
This is ugly. Can't we get `numpy.isclose()` get to work? And don't use `math` if we already have numpy.

If you provide me with a minimal example  where the old code didn't work, I'll have a look. (Basically, add a *test* that fails with the old numpy-based code)